nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -254,6 +254,8 @@ def process_full_tracks(df_all, thresh, pix_size, interval):
     df_full = add_growth_features.add_early_growth_rate(df_full, interval)
     df_full = add_growth_features.add_late_growth_rate_by_endpoints(df_full)
     df_full = add_growth_features.fit_tracks_to_time_powerlaw(df_full, "volume", interval)
```

Comment:
Can we make this call `fit_tracks_to_model` instead? Fit `tracks_to_time_powerlaw` seems like it is only needed in case we want to get the `alpha` for surface area correct? But it looks like we are not even running it for SA...except for maybe somewhere in the supp (not totally sure)?

Reply:
Yup you're right, the new `fit_tracks_to_model` creates a more general version of what `fit_tracks_to_time_powerlaw` did and can make it obsolete